www/src/content/docs/docs/get-started.mdx

```diff
@@ -1,86 +1,319 @@
+    }),
+
+    name: z.string().min(1).max(255).openapi({
+      description: "The user's full name",
+      example: "Matthew"
+    }),
+
+    email: z.string().email().openapi({
+      description: "The user's email address",
+      example: "matthew@cloudflare.com"
+    })
+  })
+  .openapi({
+    ref: "User",
+    description: "Schema for user response including ID"
+  });
+
+const CreateUserSchema = UserResponseSchema.omit({ id: true }).openapi({
+  ref: "CreateUser",
+  description: "Schema for creating a new user"
+});
+```
+
+<Aside type="note">
+  `.extend` does not propagate the openapi metadata to the new schema. Instead
+  use `.merge` .
+</Aside>
+Next add a description to the route:
+
+```ts
+import { describeRoute } from "hono-openapi";
+import { resolver, validator as zValidator } from "hono-openapi/zod";
+
+app.post(
+  "/api/user",
+  describeRoute({
+    description: "Create a new user",
+    requestBody: {
+      content: {
+        "application/json": {
+          schema: resolver(CreateUserSchema)
+        }
+      }
+    },
+    responses: {
+      201: {
+        description: "The created user",
+        content: {
+          "application/json": {
+            schema: resolver(UserResponseSchema)
+          }
+        }
+      }
+    }
+  }),
+  zValidator("json", CreateUserSchema),
+  async (c) => {
+    const { name, email } = c.req.valid("json");
+
+    // define what the route handler does
+
+    return c.json(newUser);
+  }
+);
+```
+
+Make the OpenAPI spec available:
+
+```ts
+import { openAPISpecs } from "hono-openapi";
+app.get(
+  "/openapi",
+  openAPISpecs(app, {
+    documentation: {
+      info: {
+        title: "Hono",
+        version: "1.0.0",
+        description: "API for users"
+      }
+    },
+    servers: [
+      {
+        url: "http://localhost:8787",
+        description: "Local server"
+      }
+    ]
+  })
+);
+```
+
+</details>
+
+<details>
+<summary>Fiberplane's `createOpenAPISpec` helper function</summary>
+
+The other option is to integrate OpenAPI documentation into an existing Hono codebase. Refactoring the application to use Hono Zod OpenAPI or Rhinobase provides fine-grained documentation, offering long-term benefits, but requires more effort. For a quicker solution, Fiberplane provides a helper function to generate an OpenAPI spec from the existing code.
+
+{" "}
+<PackageManagers type="add" pkg="@fiberplane/hono" />
+
+This step involves importing the `createOpenAPISpec` function from Fiberplane and creating an endpoint at `/openapi.json` to generate and return an OpenAPI spec.
+
+```ts
+import { createOpenAPISpec } from "@fiberplane/hono";
+
+app.get("/openapi.json", (c) => {
+  const spec = createOpenAPISpec(app, {
+    info: { title: "My API", version: "1.0.0" }
+  });
+  return c.json(spec);
+});
+```
+
+</details>
+
+### 3. Add Fiberplane Middleware
+
+With the OpenAPI spec available, the Fiberplane middleware can now be installed and pointed to the OpenAPI spec.
```

Comment:
reworded to avoid passive voice

```suggestion
Now that the OpenAPI spec is available, mount the Fiberplane middleware and point it to the OpenAPI spec.
```